routing/datasource.go

```diff
@@ -389,12 +394,12 @@ func receiveRouteMatcher(o Options, out chan<- *matcher, quit <-chan struct{}) {
 			for _, err := range errs {
 				o.Log.Error(err)
 			}
-
-			mout = m
+			// TODO: What about the errors? Should we exclude the routes which has errors?
```

Comment:
i think we should show first the applied routes, so yes. In another wave, we can add the routes that have errors.